grc/business_logic/data_structures/application_data.py

```diff
@@ -70,6 +70,8 @@ def section_status_medical_reports(self) -> ListStatus:
         if self.need_medical_reports:
             if len(self.uploads_data.medical_reports) == 0:
                 return ListStatus.NOT_STARTED
+            elif len([file for file in self.uploads_data.medical_reports if file.password_required]) > 0:
+                return ListStatus.ERROR
             elif any_duplicate_aws_file_names(self.uploads_data.medical_reports):

```

Comment:
It would be nice to create a method here with a clear name to reduce duplicate code
(same as we currently do for `any_duplicate_aws_file_names`)

e.g.
```python
def any_password_protected_files(uploads_files: List[EvidenceFile]):
    password_protected_files = [file for file in uploads_files if file.password_required]
    return len(password_protected_files) > 0
```

then this line can say:
```python
elif any_password_protected_files(self.uploads_data.medical_reports):
```